pkg/framework/machinesets.go

```diff
@@ -34,20 +34,24 @@ import (
 // MachineSetParams represents the parameters for creating a new MachineSet
 // resource for use in tests.
 type MachineSetParams struct {
-	Name         string
-	Replicas     int32
-	Labels       map[string]string
-	Taints       []corev1.Taint
-	ProviderSpec *machinev1.ProviderSpec
+	Name                       string
+	Replicas                   int32
+	Labels                     map[string]string
+	Taints                     []corev1.Taint
+	ProviderSpec               *machinev1.ProviderSpec
+	MachinesetAuthoritativeAPI machinev1.MachineAuthority
+	MachineAuthoritativeAPI    machinev1.MachineAuthority
 }
 
 const (
-	machineAPIGroup = "machine.openshift.io"
-	Amd64           = "amd64"
-	ArchLabel       = "e2e.openshift.io/arch"
-	labelsKey       = "capacity.cluster-autoscaler.kubernetes.io/labels"
-	ReasonKey       = "machine.openshift.io/reason"
-	ReasonE2E       = "actuator-e2e"
+	machineAPIGroup            = "machine.openshift.io"
+	Amd64                      = "amd64"
+	ArchLabel                  = "e2e.openshift.io/arch"
+	labelsKey                  = "capacity.cluster-autoscaler.kubernetes.io/labels"
+	ReasonKey                  = "machine.openshift.io/reason"
+	ReasonE2E                  = "actuator-e2e"
+	MachinesetAuthoritativeAPI = machinev1.MachineAuthorityMachineAPI
+	MachineAuthoritativeAPI    = machinev1.MachineAuthorityMachineAPI
 )
```

Comment:
How about remove these two lines? Too many MachinesetAuthoritativeAPI and MachineAuthoritativeAPI looks complicated.